cypress/integration/views/common.view.ts

```diff
@@ -40,3 +40,11 @@ export const prevPageButton = "button[aria-label='Go to previous page']";
 export const lastPageButton = "button[aria-label='Go to last page']";
 export const firstPageButton = "button[aria-label='Go to first page']";
 export const pageNumInput = "input[aria-label='Current page']";
+
+//React Selectors
+export const reactSelectorUserPerspectiveSelectorWhenAdminIsSelected = {
```

Comment:
Amos, this name looks TOO LONG, very hard to read it. Can you please change the name to make it more readable?

Reply:
i like it when the param explains itself. but sure! i will change it